frontend/components/MainPage/sections/NeedsFundsSection.tsx

```diff
@@ -21,16 +24,19 @@ const FundingValue = styled.div`
 `;
 
 export const MainNeedsFunds = () => {
+  const { selectedStakingProgramId } = useStakingProgram();
+
   const {
     hasEnoughEthForInitialFunding,
     hasEnoughOlasForInitialFunding,
     serviceFundRequirements,
     isInitialFunded,
     needsInitialFunding,
-  } = useNeedsFunds();
+  } = useNeedsFunds(selectedStakingProgramId);
```

Comment:
So the active one is the one that comes from BE and when the service exists
The default one is the one we use after initial onboarding or when changing the staking contract (before it's reflected on BE)
The selected one is active ?? default 🙃

Reply:
Would probably be better to use the default one for main page though, we shouldn't see this alert once the service is created, because if it's created it means we'd had all the funds needed

Reply:
basically `active ?? default`
but waits for active to be fetched first..
`isActiveFetched ? active ?? default : null`